src/view/navigation_tree.js

```diff
@@ -162,4 +162,28 @@ class NavigationTree {
     $('#navigationTree').jstree().create_node('#' ,
       {'id': blockType, 'text': blockType }, 'last', null);
   }
+
+  /**
+   * Creates menu for right click functionality.
+   * @return {!Object} The right click menu for the nodes in the tree.
+   * //TODO: add right click functionality to tree
+   */
+   createMenu() {
+    const items = {
+      renameElement : {
+        label: "Rename",
+        action: //TODO: add function for renaming node and associated element
```

Comment:
Does this parse without a value for action?  I'd at least put in a null value.  Better yet, a () => Console.log(..).

Same below.